pkg/apis/apis.go

```diff
@@ -36,6 +37,7 @@ var (
 	// AddToScheme may be used to add all resources defined in the project to a Scheme
 	AddToScheme = Builder.AddToScheme
 	Settings    = []settings.Injectable{&settings.Settings{}}
+	Options     = []options.Injectable{&options.Options{}}
```

Comment:
Does it make sense for this to be in `apis` or can we store the options in options like we do for the `scheme`? I'm thinking `aws/karpenter` will just do `coreoptions.Options = append(coreoptions.Options, options.Options)`

Reply:
Probably not, thats a good callout. I had put it there because I was copying the injectable pattern for settings but that's not an appropriate location. The only problem is the name conflict with the `Options` struct but I think it would be reasonable to rename it to `options.Injectables`.